http/src/main/resources/init-resources/startup.sh

```diff
@@ -98,39 +113,32 @@ function validateCert() {
 
     if [ "$certFileDirectory" = "/etc" ]
     then
-      ${dockerCompose} -f /etc/proxy-docker-compose.yaml restart &> /var/start_output.txt || EXIT_CODE=$?
+      ${DOCKER_COMPOSE} -f /etc/proxy-docker-compose.yaml restart &> /var/start_output.txt || EXIT_CODE=$?
     else
-      ${dockerCompose} -f /var/docker-compose-files/proxy-docker-compose-gce.yaml restart &> /var/start_output.txt || EXIT_CODE=$?
+      ${DOCKER_COMPOSE} -f /var/docker-compose-files/proxy-docker-compose-gce.yaml restart &> /var/start_output.txt || EXIT_CODE=$?
     fi
 
     failScriptIfError ${GSUTIL_CMD}
     retry 3 ${GSUTIL_CMD} -h "x-goog-meta-passed":"true" cp /var/start_output.txt ${START_USER_SCRIPT_OUTPUT_URI}
   fi
 }
 
-FILE=/var/certs/jupyter-server.crt
-if [ -f "$FILE" ]
-then
-    CERT_DIRECTORY='/var/certs'
-    DOCKER_COMPOSE_FILES_DIRECTORY='/var/docker-compose-files'
-    GSUTIL_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gsutil'
-    GCLOUD_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gcloud'
-    DOCKER_COMPOSE='docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /var:/var docker/compose:1.29.1'
-
-    validateCert ${CERT_DIRECTORY} ${GSUTIL_CMD} ${DOCKER_COMPOSE}
-else
-    CERT_DIRECTORY='/certs'
-    DOCKER_COMPOSE_FILES_DIRECTORY='/etc'
-    GSUTIL_CMD='gsutil'
-    GCLOUD_CMD='gcloud'
-    DOCKER_COMPOSE='docker-compose'
-
-    validateCert ${CERT_DIRECTORY} ${GSUTIL_CMD} ${DOCKER_COMPOSE}
-fi
+validateCert ${CERT_DIRECTORY}
 
 JUPYTER_HOME=/etc/jupyter
 RSTUDIO_SCRIPTS=/etc/rstudio/scripts
 
+### This is only needed for gce. But doesn't hurt to have it either for non gce VMs
+#sleep 5s
+#ls -la /mnt/disks/work
+#mkdir -p /mnt/disks/work
+#chmod a+rwx /mnt/disks/work
+
+fsck.ext4 -tvy /dev/sdb
+mkdir -p /mnt/disks/work
+mount -t ext4 -O discard,defaults /dev/sdb /mnt/disks/work
```

Comment:
this `mount` should probably not happen for Dataproc, right?

Reply:
ah yea...I just wanted to see if this fixes the mount flakiness first for gce